### PR TITLE
Add quickDocs

### PR DESCRIPTION
**Pull request by mdh34**
Developer documentation app originally built for elementary - patched theme to keep the application consistent across distributions

### CODE REVIEW

**Comment by nedrichards** [comment]
bot, build com.github.mdh34.quickdocs

File: com.github.mdh34.quickdocs.json [hunk 1]
```diff
@@ -0,0 +1,75 @@
+{
+    "app-id": "com.github.mdh34.quickdocs",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.26",
```
Comment (barthalion) [inline com.github.mdh34.quickdocs.json]: Does it work with 3.28?

File: com.github.mdh34.quickdocs.json [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+    "sdk": "org.gnome.Sdk",
+    "command": "com.github.mdh34.quickdocs",
+    "finish-args": [
+        "--share=ipc",
+        "--share=network",
+        "--device=dri",
+        "--socket=x11",
+        "--socket=wayland",
+        "--talk-name=ca.desrt.dconf",
+        "--filesystem=home",
```
Comment (barthalion) [inline com.github.mdh34.quickdocs.json]: Does it need access to home?

Comment (mdh34) [reply com.github.mdh34.quickdocs.json]: There are some files in .config but I'll make it more explicit

Comment (barthalion) [reply com.github.mdh34.quickdocs.json]: If it's just for config files, there's no need to do anything. By default $XDG_CONFIG_HOME points to place accessible to the application.

Comment (mdh34) [reply com.github.mdh34.quickdocs.json]: Ok, I've updated the app to properly use the XDG directories now so this isn't needed :)

File: com.github.mdh34.quickdocs.json (hunk 1 above)
Comment (nedrichards) [reply com.github.mdh34.quickdocs.json]: My memory has been that elementary apps like to match the versions of libraries etc. on the elementary platform, so they often have older runtimes (which is fine! we want to make sure things use the tested set). However obviously the newer is the 'best' as we'd like to reduce version skew and bandwidth and storange usage for end users.

**Comment by nedrichards** [comment]
bot, build com.github.mdh34.quickdocs

File: com.github.mdh34.quickdocs.json (hunk 1 above)
Comment (mdh34) [reply com.github.mdh34.quickdocs.json]: I've been building this on the juno beta which uses the 3.28 stack so it isn't a problem :)

File: com.github.mdh34.quickdocs.json [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+                    "type":"archive",
+                    "url":"https://gitlab.gnome.org/GNOME/devhelp/-/archive/3.28.1/devhelp-3.28.1.tar.gz",
+                    "sha256":"71de1f7ad5f574772fa401f7278ad878d52e374c2940fb2f2f049eb060e7fe08"
+                }
+            ]
+        },
+
+        {
+            "name": "quickdocs",
+            "buildsystem": "meson",
```
Comment (TingPing) [inline com.github.mdh34.quickdocs.json]:
```json
"config-opts": ["--buildtype=release"]
```

Comment (mdh34) [reply com.github.mdh34.quickdocs.json]: added :)

**Comment by barthalion** [comment]
bot, build com.github.mdh34.quickdocs

**Comment by nedrichards** [comment]
Repository has been created: https://github.com/flathub/com.github.mdh34.quickdocs

**Comment by mdh34** [comment]
Great, thanks!

**Pull request closed without merging**